### PR TITLE
Style landing page

### PR DESCRIPTION
**Pull request by msachi**
related #24

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -1 +1,38 @@
-(function() {})()
+$(".dropdown-menu li a").click(function(){
```
Comment (des-des) [inline js/app.js]: Wrap this file in function so you do not pollute global scope!

Comment (des-des) [reply js/app.js]: Or better, when I last used jquery (a long time ago) you would put everything in something like `$(document).ready(() => {})` (I think)

**Review by des-des — CHANGES_REQUESTED** [review]

**Comment by msachi** [comment]
Cheers - fixed! ☺️

**Comment by des-des** [comment]
:cake:

**Pull request merged**